ui/webui/src/components/storage/StorageConfiguration.jsx

```diff
@@ -125,7 +125,7 @@ const scenarios = [{
     initializationMode: 0,
     buttonLabel: _("Apply mount point assignment and install"),
     buttonVariant: "danger",
-    screenWarning: "",
+    screenWarning: "To prevent loss, make sure to backup your data.",
```

Comment:
Was not sure if we want to show it here as well but the original alert was shown already so good enough I think?

Reply:
I think as well it's good as you have it now.